src/search_count_report.py

```diff
@@ -0,0 +1,47 @@
+import csv
+from datetime import datetime
+
+from src.report import Report
+import src.utils as utils
+
+class SearchCountReport(Report):
+
+    def generate(self):
+        # Write fields to search_count.csv
+        self.writer.writerow(['day', 'hour', 'count'])
+
+        time_to_count_dict = {}
+
+        # Read routes.csv
+        for index, row in enumerate(self.reader):
```

Comment:
you can actually do `for index, [timestamp, _] in enumerate...`, if the only variable you're accessing is `timestamp`